lib/ret_web/controllers/api/v1/account_search_controller.ex

```diff
@@ -0,0 +1,16 @@
+defmodule RetWeb.Api.V1.AccountSearchController do
+  use RetWeb, :controller
+
+  alias Ret.{Account}
+  alias RetWeb.Api.V1.{AccountView}
+
+  # Account lookup, is a POST because lookup contains sensitive information
+  def create(conn, %{"email" => email}) do
+    with %Account{} = account <- Account.account_for_email(email) do
+      record = Phoenix.View.render(AccountView, "show.json", account: account)
+      conn |> send_resp(200, %{records: [record]} |> Poison.encode!())
+    else
+      _ -> conn |> put_status(:not_found) |> halt()
```

Comment:
Would it make sense to respond with an `{errors: ...}` json in this case as well? It's generally nicer for an API to always return json if it normally returns json.

Reply:
i think for the 404's this seems right. we're already `halt()`ing for 401s in other places, but i'll update the 404 paths.